dev-qt/qtdatavis3d/qtdatavis3d-5.7.0.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit qt5-build
+
+DESCRIPTION="OpenGL data visualisation module for the Qt5 framework"
+
+if [[ ${QT5_BUILD_TYPE} == release ]]; then
+	KEYWORDS="~amd64 ~x86"
+fi
+
+IUSE="gles2"
+
+DEPEND="
+	~dev-qt/qtcore-${PV}
+	~dev-qt/qtdeclarative-${PV}[gles2=]
+	~dev-qt/qtgui-${PV}[gles2=]
+	~dev-qt/qtnetwork-${PV}
+"
+RDEPEND="${DEPEND}"
+
+src_configure() {
+	local myconf=(
```

Comment:
`myconf` is only meaningful in qtbase modules. The `[gles2=]` USE dep should be enough.